### ISSUE TITLE
Feature Request: Separate Navigation Cursor Size Setting

### ISSUE THREAD

**Issue by drtbyk** [osmandapp/OsmAnd-iOS#2517]
Currently, the Navigation Cursor Size is linked to the Text Size setting.  This is problematic when selecting a Text Size  >100% as the Navigation Cursor is now so large as to block the view of small towns completely.  This is most evident when using CarPlay.

**Feature Request:** Separate Navigation Cursor Size setting from Text Size Setting.

**Comment by GPSFietser** [comment]
I agree. The same goes for the POI and Favorite icons size. They are also linked to the text size.

**Feature request**: Seperate size setting for
- Text
- Position Cursor
- POI/Favorite Icons

**Comment by vshcherb** [comment]
@Zirochkabila Screenshots are needed to make a right decision

**Comment by Zirochkabila** [comment]
@drtbyk @jsgblom Please clarify:
1. The device you use, OS version
2. Application version
3. A short video and screenshots where we could see the error

**Comment by drtbyk** [comment]
1. iPhone 14 Pro Max, iOS v16.3.12. OsmAnd v4.4.5.43. Screenshots:Cheers,Dan T.Sent from my iPad ProOn Feb 15, 2023, at 02:15, Anatolii ***@***.***> wrote:﻿
@drtbyk @jsgblom Please clarify:

The device you use, OS version
Application version
A short video and screenshots where we could see the error

**Comment by jsgblom** [comment]
> @drtbyk @jsgblom Please clarify:
> 
> 1. The device you use, OS version
> 2. Application version
> 3. A short video and screenshots where we could see the error

This issue is relevant to both Android- and iOS-OsmAnd, on small (iPhone 13 mini) and big screens (Samsung Cover Pro). since as long as I can remember (many years). 

In my opinion the default text size used for streetnames is too big, plus the names of minor roads (tertiary and smaller) are displayed too soon (which is a different matter/issue), as a result cluttering the map.

To solve this I could use a text size of 75%, but then the position arrow and the POI's become too small. On the other hand, as I cyclist I would prefer the arrow a little bigger, but this is only possible through setting the text size bigger, making the streetnames and POI icons way too big. See attached screenshots of my iPhone 13 mini and Xcover Pro.

BTW: Most of the time I'm not interested in streetnames. I think the streetnames are displayed too soon, I would prefer to display them at a higher zoom level.

[osmand-ios_text-size-issue.pdf](https://github.com/osmandapp/OsmAnd-iOS/files/10766546/osmand-ios_text-size-issue.pdf)

[osmand-android_text-size-issue.pdf](https://github.com/osmandapp/OsmAnd-iOS/files/10766548/osmand-android_text-size-issue.pdf)

**Comment by Zirochkabila** [comment]
The question is quite subjective. For me, 100% is the most balanced option. In all the others, you can change the text size. 
**The only thing is that we can add the ability to resize the Position arrow** 

1. **75%** Position arrow and POI’s too small
2. **100%** Street names too big, already cluttering the map. POI’s OK, but I personally would prefer a bigger position arrow.
3. **125%** Street names way too big, cluttering the map. POI’s a little too big, Position arrow OK.

<details><summary>75%</summary>
<img width="240" alt="Screenshot 2023-01-31 at 11 19 41" src="https://user-images.githubusercontent.com/104760013/219867234-640c87ba-1881-4795-80de-7bef52b79ef6.jpeg">
<img width="400" alt="Screenshot 2023-01-31 at 11 19 41" src="https://user-images.githubusercontent.com/104760013/219867236-97785ba8-0b0a-453d-aa7f-a229d5f9c953.jpeg">
</details> 

<details><summary>100%</summary>
<img width="240" alt="Screenshot 2023-01-31 at 11 19 41" src="https://user-images.githubusercontent.com/104760013/219867304-cf5502f8-d59e-406d-86f3-1e3b0bf92f37.jpeg">
<img width="400" alt="Screenshot 2023-01-31 at 11 19 41" src="https://user-images.githubusercontent.com/104760013/219867309-d2e32443-017e-4df4-b4d1-465e6d451311.jpeg">
</details> 

<details><summary>125%</summary>
<img width="240" alt="Screenshot 2023-01-31 at 11 19 41" src="https://user-images.githubusercontent.com/104760013/219867372-765b8e51-547c-49cf-a913-bd08df79735d.jpeg">
<img width="400" alt="Screenshot 2023-01-31 at 11 19 41" src="https://user-images.githubusercontent.com/104760013/219867378-d3fce12a-c62a-4849-ad66-33ea3c05118c.jpeg">
</details>

**Comment by jsgblom** [comment]
> The question is quite subjective. For me, 100% is the most balanced option. In all the others, you can change the text size. **The only thing is that we can add the ability to resize the Position arrow**
> 
> 1. **75%** Position arrow and POI’s too small
> 2. **100%** Street names too big, already cluttering the map. POI’s OK, but I personally would prefer a bigger position arrow.
> 3. **125%** Street names way too big, cluttering the map. POI’s a little too big, Position arrow OK.

@Zirochkabila: What do you mean by **The only thing is that we can add the ability to resize the Position arrow**. Why not the size of the POI's, since their size is also important.

For me, the setting **text size** as it is implemented now is useless because of the side effects (POI / Position arrow). The only 'workable' option is 100% IMHO.

One more thing with the text size: I think there are too many options 150% and upwards makes the map look very messy and cluttered! On Android there is an option of 33% which makes the tekst unreadable.

**Comment by jsgblom** [comment]
When I make the text smaller the streetnames are still readable to me, but the icons in POI I find harder to 'read'. So the 'readable' size of streetnames and POI icons is not directly related. That's why I think a separate setting would be very helpful.

Furthermore, the settings "text size" might be confusing/surprising since a user would probably not expect that the size of the Position arrow and POI icons are changed along with it.

**Comment by drtbyk** [comment]
Separating the setting for Text Sizing from POI/ICON's would provide the best options for Accessibility. Not all users have the same eyesight and/or needs for UI interaction.
